src/webview/android/binding.rs

```diff
@@ -57,14 +57,30 @@ fn handle_request(env: JNIEnv, request: JObject) -> Result<jobject, JniError> {
   if let Some(handler) = REQUEST_HANDLER.get() {
     let final_request = match request_builder.body(Vec::new()) {
       Ok(req) => req,
-      Err(_) => {
+      Err(e) => {
+        log::warn!("Failed to build response: {}", e);
         return Ok(*JObject::null());
       }
     };
     let response = (handler.0)(final_request);
     if let Some(response) = response {
-      let status_code = response.status().as_u16() as i32;
-      let reason_phrase = "OK";
+      let status = response.status();
+      let status_code = status.as_u16();
+      let status_err = if status_code < 100 {
+        Some("Status code can't be less than 100")
+      } else if status_code > 599 {
+        Some("statusCode can't be greater than 599.")
+      } else if status_code > 299 && status_code < 400 {
+        Some("statusCode can't be in the [300, 399] range.")
```

Comment:
Why 300+ are not allowed?

Reply:
I guess Android can't handle redirects. I just took this from their own code, see the link in the PR description.

Reply:
should probably document that with a link to their docs